src/cmd/flux-job.c

```diff
@@ -104,11 +108,18 @@ static struct optparse_subcommand subcommands[] = {
       NULL,
     },
     { "cancel",
-      "[OPTIONS] id ...",
-      "Abort job(s)",
-      cmd_cancel,
+      "[OPTIONS] id [message ...]",
+      "Alias for flux job raise",
+      cmd_raise,
```

Comment:
I think it is ok to hid the fact that `flux job cancel` is an alias for `flux job raise` and provide help message that says "Cancel a job" for better UI.

I love that you can provide a message when you cancel a job.

Reply:
Sounds good.